pkg/merger/BUILD.bazel

```diff
@@ -11,6 +11,7 @@ go_library(
         "//util/gcs:go_default_library",
         "//util/metrics:go_default_library",
         "@com_github_golang_protobuf//proto:go_default_library",
+        "@com_google_cloud_go_storage//:go_default_library",
```

Comment:
move this one line lower

Reply:
`bazel run @io_k8s_repo_infra//hack:update-bazel` will do this for you